src/plugins/intel_cpu/thirdparty/CMakeLists.txt

```diff
@@ -174,8 +174,22 @@ if(ENABLE_MLAS_FOR_CPU)
     ov_install_static_lib(mlas ${OV_CPACK_COMP_CORE})
 endif()
 
-if(ENABLE_SHL_FOR_CPU)
+function(ov_build_shl_static)
```

Comment:
Could you please add the comment why we have to add these flags?

And is it possible to move this function definition in body of `if(ENABLE_SHL_FOR_CPU)`?